frontend/src/routes/index.ts

```diff
@@ -23,6 +25,11 @@ const routes: Routes = {
     path: '/',
     title: 'PermaPlanT',
   },
+  [Pages.DrawingPage]: {
+    component: DrawingPage,
+    path: '/drawing-page',
```

Comment:
And please also add a link from landing page (rebase first):

```suggestion
    path: '/map',
```